ArduCopter/ArduCopter.cpp

```diff
@@ -324,6 +324,9 @@ void Copter::fourhundred_hz_logging()
     if (should_log(MASK_LOG_ATTITUDE_FAST)) {
         Log_Write_Attitude();
     }
+#if FRAME_CONFIG == HELI_FRAME
```

Comment:
That's a whole lot of logging....

Reply:
this was only to do troubleshooting and verify the FFT was being done correctly.  I had problems getting the correct RPM and couldn't figure it out.  I put in a PR at Tridge's request so he could help me sort this out.  This will commit will be removed before merging.